data/settings.js

```diff
@@ -7358,6 +7358,19 @@ Configures when passdbs should be skipped:
 | \`unauthenticated\` | Skip if user hasn't yet been successfully authenticated by the previous passdbs. |`
 	},
 
+	passdb_use_worker: {
+		tags: [ 'passdb' ],
+		values: setting_types.BOOLEAN,
+		default: 'no',
+		text: `
+If \`yes\`, run the passdb lookup in auth-worker process instead of the main
+auth process.
+
+::: info
+This setting may be overridden by some passdbs.
```

Comment:
I feel the current markup of the "Default" value is not optimal, because it says: `[None]`, when it's not given. I think it could be an object that either gives the given value (explicit default) or you can setup, whether it should be given at all (no value) or a default with an explicit exception.

Anyway I didn't want to spend more time than necessary on the settings framework, because then I would still be porting 100 commits or so.

Reply:
In any case the info text needs changing to something like "The setting defaults may be different in some passdbs."